ethereum/check_swap_creator_contract_test.go

```diff
@@ -141,7 +141,8 @@ func TestSepoliaContract(t *testing.T) {
 	if errors.Is(err, errInvalidSwapCreatorContract) && sepoliaKey != "" {
 		pk, err := ethcrypto.HexToECDSA(sepoliaKey) //nolint:govet // shadow declaration of err
 		require.NoError(t, err)
-		forwarderAddr := common.StagenetConfig().ForwarderAddr
+		forwarderAddr, err := DeployGSNForwarderWithKey(context.Background(), ec, pk)
+		require.NoError(t, err)
```

Comment:
This deploys a forwarder contract instead of using the hardcoded `ForwarderAddr`

Reply:
See the comments I added to the ticket. Deploying contracts on `mainnet` is expensive and the forwarder contract is already deployed by its creators, so it probably doesn't make sense to ever deploy it except in ganache. It is cheap to deploy it on sepolia, but we should treat `stagenet` as close as possible to the way we treat `mainnet` and not plan to redeploy it ever again on sepolia, so that code can be removed. It was just convenience code code for us and only ever executed if you were running the unit test by hand with a key set.